pkg/arvo/sys/lull.hoon

```diff
@@ -1870,7 +1870,12 @@
         [%public-keys =public-keys-result]            ::  ethereum changes
         [%turf turf=(list turf)]                      ::  domains
     ==                                                ::
-  ::  +seed: private boot parameters
+  ::  +feed: potential boot parameters
```

Comment:
Is feed used?

Reply:
In `lib/vere` (and associated tests), which is exclusively used by the runtime via the ivory pill.